### DISCUSSION TITLE
Welcome to Does It ARM Discussions!

### DISCUSSION THREAD

**Discussion by ThatGuySam** [ThatGuySam/doesitarm#403] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Share app updates and info
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Muadlib** [comment]
Hi I have a question on on the listing for Acrobat Reader being fully native.  If the app is fully native, but the install tool required to install the app isn't, should get the app get full 'passing marks' on the list?  I think Reader is the only app I've encountered out of the 100 or so apps I've installed where a fully native product still requires the user to install Rosetta.

**Comment by ThatGuySam** [comment]
@Muadlib You make a good point. 

This is why I’ve been steering towards the wording “Native Apple Silicon Support” since “Fully Native” might imply support details that we’re not aware of. 

I think “Native Apple Silicon Support” still applies to Acrobat Reader for the following reasons: 
- it’s still running Natively to Apple Silicon for 99% of the time you’ll be using
- Adobe will update the installer eventually
- Most Mac users have or will have to enable Rosetta for a few more years. 

We can also add a note that the installer requires Rosetta. 

However, I like to champion discussion so I want to hear additional thoughts or criticisms of that thinking.

**Comment by Muadlib** [comment]
Thanks!  and thanks for the site (and keeping it current).  I check weekly
to see what apps we use in our shop are now native (and whether we can
finally remove Rosetta from our deploy list).  Super useful resource!

**Reply by ThatGuySam** [reply]
Very welcome! Glad to hear it's helping people! 

Feel free to reach out if you run into any issues or have questions.

**Comment by alelg** [comment]
Hello!

Thank you for this website, it is very useful!
How and who can we notify about software update?
Virtualbox 7.0.4 seems natively fit for mac ARM and not tagged as it.

**Reply by ThatGuySam** [reply]
@alelg You can make an App Update Request via a New Issue: 
https://github.com/ThatGuySam/doesitarm/issues/new/choose
